### ISSUE TITLE
Leases container manual setup

### ISSUE THREAD

**Issue by shivaprathipati** [microsoft/kafka-connect-cosmosdb#142]
To get the source connector working I had to explicitly setup a leases container. For example, if I am interested in setting up a source connector for a cosmosdb container called "products", I had to manually setup a leases container called "products-leases". 

Setup instructions don't talk about setting up a leases container. When I was seeing an error in the logs, I realized the code is looking for a leases container. I believe adding an instruction to setup leases container manually will help future users of the cosmosdb connector.

Let me know if you need any additional information.

**Comment by marcelopio** [comment]
The leases code is very wrong, I fixed it by using:

```diff
Index: src/main/java/com/microsoft/azure/cosmosdb/kafka/connect/source/CosmosDBSourceTask.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/main/java/com/microsoft/azure/cosmosdb/kafka/connect/source/CosmosDBSourceTask.java	(date 1605818961553)
+++ src/main/java/com/microsoft/azure/cosmosdb/kafka/connect/source/CosmosDBSourceTask.java	(date 1605818961553)
@@ -136,6 +136,7 @@
                 .endpoint(this.settings.getEndpoint())
                 .key(this.settings.getKey())
                 .consistencyLevel(ConsistencyLevel.SESSION)
+                .contentResponseOnWriteEnabled(true)
                 .buildAsyncClient();
     }
@@ -180,16 +181,9 @@
         logger.info("Creating new lease container.");
         try {
             leaseContainerResponse = leaseCollection.read().block();
-
-        } catch (RuntimeException ex) {
+        } catch (CosmosException ex) {
             // Swallowing exceptions when the type is CosmosClientException and statusCode is 404
-            if (ex instanceof CosmosException) {
-                CosmosException cosmosClientException = (CosmosException) ex;
-
-                if (cosmosClientException.getStatusCode() == 404) {
-                    throw ex;
-                }
-            } else {
+            if (ex.getStatusCode() != 404) {
                 throw ex;
             }
```

**Comment by ryancrawcour** [comment]
would be great if you could submit this change as a pull-request @marcelopio

**Comment by ryancrawcour** [comment]
closed by #149
